solidity/test/LPRewardsTest.js

```diff
@@ -402,3 +406,120 @@ describe("LPRewards", () => {
     await wrappedToken.approve(lpRewards.address, amount, {from: staker})
   }
 })
+
+describe("LPRewardsTBTCSaddle", () => {
+  const tokenDecimalMultiplier = web3.utils.toBN(10).pow(web3.utils.toBN(18))
+
+  const keepTokenOwner = accounts[1]
+  const lpTokenOwner = accounts[2]
+
+  const lpRewardsOwner = accounts[3]
+
+  const stakerEOA = accounts[4]
+  const thirdParty = accounts[5]
+
+  let keepToken
+  let lpToken
+  let lpRewards
+  let lpRewardsStaker
```

Comment:
As we've got `stakerEOA` constant, then maybe we could name this variable `stakerContract`?

Reply:
https://github.com/keep-network/keep-ecdsa/pull/675/commits/e369bdc3fccf331e3f45906742db23ff4b3f1b47